src/main/java/com/atlassian/bitbucket/jenkins/internal/model/deployment/BitbucketCDCapabilities.java

```diff
@@ -0,0 +1,30 @@
+package com.atlassian.bitbucket.jenkins.internal.model.deployment;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Set;
+
+import static java.util.Collections.unmodifiableSet;
+import static java.util.Objects.requireNonNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketCDCapabilities {
+
+    public static final String DEPLOYMENTS = "deployments";
+
+    private final Set<String> cdCapabilities;
+
+    @JsonCreator
+    public BitbucketCDCapabilities(Set<String> cdCapabilities) {
+        this.cdCapabilities = unmodifiableSet(requireNonNull(cdCapabilities, "cdCapabilities"));
```

Comment:
```suggestion
        this.cdCapabilities = unmodifiableSet(requireNonNull(cdCapabilities, "Deployment capability missing."));
```

Reply:
We usually use the variable name as the message in calls to `requireNonNull`, and the rest of the usages in this repo do it that way too.

Reply:
ah was going off what we had in BitbucketCICapabilities#21, but happy to not change